PhysicsTools/PatAlgos/python/producersLayer1/muonProducer_cfi.py

```diff
@@ -71,7 +71,9 @@
         #        deltaR = cms.double(0.3)
         #    )),
     ),
-
+    # Read and store combined inverse beta
+    addInverseBeta    = cms.bool(False),  
```

Comment:
considering that this is a case of simply adding a new variable, I think that it should be fine to just fill it.
This should be acceptable for the "no-change" policy. (the most recent example is in #29276 )